contracts/FlowToken.cdc

```diff
@@ -1,4 +1,5 @@
 import FungibleToken from 0xFUNGIBLETOKENADDRESS
+import FungibleTokenMetadataViews from "./FungibleTokenMetadataViews.cdc"
 
 pub contract FlowToken: FungibleToken {
```

Comment:
I think the contract should implement `ViewResolver` if possible. Implies the need to add the contract interface methods to this contract.

Reply:
great point! I'll add that

Reply:
ope, nevemind, I just remembered that we can't add new interface conformances in an upgrade, so that won't be possible. I can still add the methods to the contract though